testing/tests/DevExpress.ui.widgets.scheduler/appointmentPopup.tests.js

```diff
@@ -1539,40 +1540,51 @@ QUnit.test('Multiple showing appointment popup for recurrence appointments shoul
 
 QUnit.test('Appointment popup will render even if no appointmentData is provided (T734413)', function(assert) {
     const scheduler = createInstance();
+    const currentDate = new Date(2020, 2, 4);
+    const cellDuration = 60;
+    scheduler.option('currentDate', currentDate);
+    scheduler.option('cellDuration', cellDuration);
+
     scheduler.instance.showAppointmentPopup({}, true);
     scheduler.instance.hideAppointmentPopup(true);
     scheduler.instance.showAppointmentPopup({}, true);
     const { startDate, endDate } = scheduler.appointmentForm.getFormInstance().option('formData');
     const appointmentPopup = scheduler.appointmentPopup;
 
-    assert.equal(startDate, null, 'startDate has null in the dxForm');
-    assert.equal(endDate, null, 'endDate has null in the dxForm');
+    assert.equal(startDate.getTime(), currentDate.getTime(), 'startDate is currentDate in Appointment Form');
+    assert.equal(endDate.getTime(), new Date(currentDate.getTime() + cellDuration * toMs('minute')).getTime(), 'endDate is currentDate + cellDuration in Appointment Form');
```

Comment:
It is better to compare with a specific value here instead of math from sources: 
`new Date(currentDate.getTime() + cellDuration * toMs('minute')).getTime()`